### PR TITLE
adding parent user list control

### PR DESCRIPTION
**Pull request by icreated**
implementaion of addUserRole in 2 components: Workspace & Scenario
Organization not needs it because of it parent nature

### CODE REVIEW

File: scenario/src/main/kotlin/com/cosmotech/scenario/azure/ScenarioServiceImpl.kt [hunk 1]
```diff
@@ -896,7 +898,8 @@ internal class ScenarioServiceImpl(
   ): ScenarioAccessControl {
     val scenario = findScenarioByIdNoState(organizationId, workspaceId, scenarioId)
     csmRbac.verify(scenario.getRbac(), PERMISSION_WRITE_SECURITY, scenarioPermissions)
-    csmRbac.getAccessControl(scenario.getRbac(), identityId)
+    csmRbac.getAccessControlOrThrow(
+        scenario.getRbac(), identityId, "User '$identityId' not found in scenario $workspaceId")
```
Comment (jreynard-code) [inline scenario/src/main/kotlin/com/cosmotech/scenario/azure/ScenarioServiceImpl.kt:902]: Following the dev review on cosmotech-api-common, use the correct method `(verify|check)Rights`

File: workspace/src/main/kotlin/com/cosmotech/workspace/azure/WorkspaceServiceImpl.kt [hunk 2]
```diff
@@ -381,7 +385,8 @@ internal class WorkspaceServiceImpl(
   ): WorkspaceAccessControl {
     val workspace = findWorkspaceByIdNoSecurity(organizationId, workspaceId)
     csmRbac.verify(workspace.getRbac(), PERMISSION_WRITE_SECURITY)
-    csmRbac.getAccessControl(workspace.getRbac(), identityId)
+    csmRbac.getAccessControlOrThrow(
+        workspace.getRbac(), identityId, "User '$identityId' not found in workspace $workspaceId")
```
Comment (jreynard-code) [inline workspace/src/main/kotlin/com/cosmotech/workspace/azure/WorkspaceServiceImpl.kt:389]: Following the dev review on cosmotech-api-common, use the correct method `(verify|check)Rights`

File: organization/src/main/kotlin/com/cosmotech/organization/azure/OrganizationServiceImpl.kt [hunk 3]
```diff
@@ -278,7 +278,10 @@ class OrganizationServiceImpl(private val csmRbac: CsmRbac, private val csmAdmin
   ): OrganizationAccessControl {
     val organization = findOrganizationById(organizationId)
     csmRbac.verify(organization.getRbac(), PERMISSION_WRITE_SECURITY)
-    csmRbac.getAccessControl(organization.getRbac(), identityId)
+    csmRbac.getAccessControlOrThrow(
```
Comment (jreynard-code) [inline organization/src/main/kotlin/com/cosmotech/organization/azure/OrganizationServiceImpl.kt:281]: Following the dev review on cosmotech-api-common, use the correct method `(verify|check)Rights`

**Review by jreynard-code — CHANGES_REQUESTED** [review]
Some remarks due to dev review on cosmotech-api-common

File: organization/src/main/kotlin/com/cosmotech/organization/azure/OrganizationServiceImpl.kt (hunk 3 above)
Comment (icreated) [reply organization/src/main/kotlin/com/cosmotech/organization/azure/OrganizationServiceImpl.kt:281]: done

**Pull request merged**